_security/access-control/permissions.md

```diff
@@ -65,6 +65,16 @@ Rather than individual permissions, you can often achieve your desired security
 {: .tip }
 
 
+### System permission
+
+The system permission `system:admin/system_index` is unique among other permissions in that it extends some traditional admin-only accessibility to non-admin users. This permission gives normal users the ability to modify system indexes for the cluster. It excludes, however, access to the security system index `.opendistro_security`, which is used to store the configuration YAML files and remains accessible only to admins with an admin certificate.
+
+Admin users that have the permission `restapi:admin/roles` are able to map the `system:admin/system_index` permission to users just as they would for a cluster or index permission. However, to preserve some control over this permission, the configuration setting `plugins.security.system_indices.additional_control.enabled` allows administrators to disable this permission by setting it to `false`. For more information about this setting, see [Enabling user access to system indexes]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#enabling-user-access-to-system-indexes).
+
+Keep in mind that an admin user who enables this feature necessarily accepts the risks involved with giving normal users access to system indexes, which may contain sensitive information and configurations essential to a cluster's health.
```

Comment:
We should also mention that users with permission to modify roles `restapi:admin/roles` can essentially grant themselves or any other user in cluster access to system indices. However, these permissions will have to be individually granted. They cannot be granted as a `*` permission. i.e. `system:admin/*` is not a valid permission.

Also, `.opendistro_security` index is not permission-able.

Reply:
Right, covered both these points in current text. Thanks